apps/marketplace/components/Collaborate/Landing.js

```diff
@@ -0,0 +1,298 @@
+          ).length
+          return (
+            <div>
+              <div className={`${styles.funded} row`}>
+                <span className={styles.projectCount}>{projectCount}</span> project{projectCount > 1 ? 's ' : ' '}
+                {childProps.match.params.state === 'all' ? '' : ` in ${childProps.match.params.state.toUpperCase()}`}
+                <span className={styles.sidebar}>
+                  Show:{' '}
+                  <NavLink to={`/2/collaborate/funded/all`} activeClassName={styles.activeLink}>
+                    all
+                  </NavLink>{' '}
+                  |{' '}
+                  <NavLink to={`/2/collaborate/funded/nsw`} activeClassName={styles.activeLink}>
+                    NSW
+                  </NavLink>{' '}
+                  |{' '}
+                  <NavLink to={`/2/collaborate/funded/vic`} activeClassName={styles.activeLink}>
+                    VIC
+                  </NavLink>{' '}
+                  |{' '}
+                  <NavLink to={`/2/collaborate/funded/wa`} activeClassName={styles.activeLink}>
+                    WA
+                  </NavLink>{' '}
+                  |{' '}
+                  <NavLink to={`/2/collaborate/funded/sa`} activeClassName={styles.activeLink}>
+                    SA
+                  </NavLink>{' '}
+                  |{' '}
+                  <NavLink to={`/2/collaborate/funded/qld`} activeClassName={styles.activeLink}>
+                    QLD
+                  </NavLink>{' '}
+                  |{' '}
+                  <NavLink to={`/2/collaborate/funded/tas`} activeClassName={styles.activeLink}>
+                    TAS
+                  </NavLink>{' '}
+                  |{' '}
+                  <NavLink to={`/2/collaborate/funded/act`} activeClassName={styles.activeLink}>
+                    ACT
+                  </NavLink>{' '}
+                  |{' '}
+                  <NavLink to={`/2/collaborate/funded/nt`} activeClassName={styles.activeLink}>
+                    NT
+                  </NavLink>
+                </span>
+                <br />
+                <hr />
+              </div>
+
+              {funded.map(val => {
+                if (val.state.includes(childProps.match.params.state) || childProps.match.params.state === 'all') {
+                  return (
+                    <div className={`${styles.project} row`} key={`${uniqueID()}${val.title}`}>
+                      <h3>
+                        {val.title}
+                      </h3>
+                      <div>
+                        {val.applicant}, {val.location}
+                      </div>
+                      <br />
+                      <p>
+                        {val.description}
+                      </p>
+                      <hr />
+                    </div>
+                  )
+                }
+                return undefined
```

Comment:
Is this necesary?

Reply:
eslint wants returns
  109:31  error  Expected to return a value at the end of arrow function  array-callback-return
  109:31  error  Expected to return a value at the end of arrow function  consistent-return